packages/blade/src/components/Link/Link/__tests__/Link.native.test.tsx

```diff
@@ -0,0 +1,102 @@
+import { fireEvent, waitFor } from '@testing-library/react-native';
+import { Linking } from 'react-native';
+import renderWithTheme from '../../../../_helpers/testing/renderWithTheme.native';
+import { InfoIcon } from '../../../Icons';
+import Link from '../Link';
+jest.mock('react-native/Libraries/Linking/Linking', () => ({
+  openURL: jest.fn(() => Promise.resolve()),
+  canOpenURL: jest.fn(() => Promise.resolve(true)),
+}));
+
+beforeAll(() => jest.spyOn(console, 'error').mockImplementation());
+afterAll(() => jest.restoreAllMocks());
+
+describe('<Link />', () => {
+  it('should render link with default properties', () => {
```

Comment:
aren't these tests too many? we are writing tests for baseline and then again we are testing all the P&C of this link? i think we can limit it just to write tests for link? what are some use case you thought while taking this decision?

Reply:
Yup. I feel the same that there are too many tests. Big reason to add these tests was to ensure that for any reason BaseLink breaks, Link should remain protected.